build.gradle

```diff
@@ -1,16 +1,39 @@
 plugins {
   id 'java'
+  id 'net.researchgate.release' version '2.7.0'
```

Comment:
these changes might be better in the xio build.gradle - `xio-core/build.gradle`

Reply:
don't we want to run this plugin for both `xio-core` and `xio-test`?